src/main/java/com/glencoesoftware/bioformats2raw/Converter.java

```diff
@@ -487,6 +494,9 @@ public void convert()
       Memoizer memoizer;
       try {
         reader = (IFormatReader) readerClass.getConstructor().newInstance();
+        if (reader instanceof MiraxReader) {
+          ((MiraxReader) reader).setFillValue(fillValue);
```

Comment:
I know it says so in the doc string but maybe fail fast if fillValue is set and this isn't a MiraxReader?

Reply:
See b1efcd2.  For right now we just care about .mrxs data, but long term the general idea would be:

- push ```setFillValue``` and ```getFillValue``` (or whatever we agree the API should be) up to ```IFormatReader``` and ```FormatReader```
- replace ```Arrays.fill(..., (byte) 0)``` with ```Arrays.fill(..., getFillValue())``` in appropriate readers
- revert b1efcd2 and call ```setFillValue``` on all readers